### PR TITLE
Proofread Batch and Deferred Credential Endpoint sections

### PR DESCRIPTION
**Pull request by selfissued**
Correct issues identified while proofreading the Batch Credential Endpoint and Deferred Credential Endpoint sections.

### CODE REVIEW

**Review by peppelinux — APPROVED** [review]

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 1]
```diff
@@ -1077,11 +1077,11 @@ Authorization: BEARER czZCaGRSa3F0MzpnWDFmQmF0M2JW
 
 ## Batch Credential Response {#batch-credential_response}
 
-A successful Batch Credential Response MUST contain all the requested Credentials. The Batch Credential Response MUST be sent as a JSON-encoded object using the `application/json` media type.
+A successful Batch Credential Response MUST contain all the requested Credentials. The Batch Credential Response MUST be sent as a JSON object using the `application/json` media type.
 
 The following claims are used in the Batch Credential Response:
```
Comment (bc-pi) [inline openid-4-verifiable-credential-issuance-1_0.md:1082]:
```suggestion
The following parameters are used in the Batch Credential Response:
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 2]
```diff
@@ -1034,11 +1034,11 @@ The Client can request issuance of multiple Credentials of certain types and for
 
 ## Batch Credential Request {#batch-credential_request}
 
-The Batch Credential Endpoint allows a Client to send multiple Credential Request objects (see (#credential_request)) to request the issuance of multiple Credentials at once. A Batch Credential Request MUST be sent as a JSON-encoded object using the `application/json` media type.
+The Batch Credential Endpoint allows a Client to send multiple Credential Request objects (see (#credential_request)) to request the issuance of multiple Credentials at once. A Batch Credential Request MUST be sent as a JSON object using the `application/json` media type.
 
-The following keys are used in the Batch Credential Request:
+The following claims are used in the Batch Credential Request:
```
Comment (bc-pi) [inline openid-4-verifiable-credential-issuance-1_0.md:1039]: These aren't claims - this is just plain JSON in an HTTP request. 

```suggestion
The following parameters are used in the Batch Credential Request:
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 3]
```diff
@@ -1134,27 +1134,27 @@ The Batch Credential Endpoint MUST respond with an HTTP 400 (Bad Request) status
...
+The Wallet MUST present an Access Token to the Deferred Endpoint that is valid for the issuance of the Credential previously requested at the Credential Endpoint or the Batch Credential Endpoint.
 
 Communication with the Deferred Credential Endpoint MUST utilize TLS. 
 
 ## Deferred Credential Request {#deferred-credential_request}
 
 The Deferred Credential Request is an HTTP POST request. It MUST be sent using the `application/json` media type.
 
-The following claims are used in the Batch Credential Response:
+The following claims are used in the Batch Credential Request:
```
Comment (bc-pi) [inline openid-4-verifiable-credential-issuance-1_0.md:1153]:
```suggestion
The following parameter is used in the Batch Credential Request:
```

**Review by bc-pi — CHANGES_REQUESTED** [review]

**Review by bc-pi — APPROVED** [review]

**Review by tplooker — APPROVED** [review]

**Review by awoie — APPROVED** [review]

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 4]
```diff
@@ -1134,27 +1134,27 @@ The Batch Credential Endpoint MUST respond with an HTTP 400 (Bad Request) status
...
+When the Credential Issuer requires `proof` objects to be present in the Batch Credential Request, but does not receive them, it will return a Batch Credential Error Response with a `c_nonce` using `invalid_proof` error code, as defined in (#issuer-provided-nonce).
 
 # Deferred Credential Endpoint {#deferred-credential-issuance}
 
 This endpoint is used to issue a Credential previously requested at the Credential Endpoint or Batch Credential Endpoint in cases where the Credential Issuer was not able to immediately issue this Credential. Support for this endpoint is OPTIONAL.
 
-The Wallet MUST present to the Deferred Endpoint an Access Token valid for the issuance of the Credential previously requested at the Credential Endpoint or the Batch Credential Endpoint. 
+The Wallet MUST present an Access Token to the Deferred Endpoint that is valid for the issuance of the Credential previously requested at the Credential Endpoint or the Batch Credential Endpoint.
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:1145]:
```suggestion
The Wallet MUST present to the Deferred Endpoint an Access Token that is valid for the issuance of the Credential previously requested at the Credential Endpoint or the Batch Credential Endpoint.
```
it is easier to understand when there is no confusion what `that` refers to (access token or deferred endpoint in this case) - makes it easier for non-native speakers from languages not similar to english-family languages like japanese

Comment (selfissued) [reply openid-4-verifiable-credential-issuance-1_0.md:1145]: Fair enough

**Pull request merged**